From 19a84495eaccd533693ed09667a3a24218d3e758 Mon Sep 17 00:00:00 2001
From: Tal Yacobi <tyacobi@habana.ai>
Date: Sun, 10 Sep 2023 17:20:34 +0300
Subject: [PATCH] [SW-0] hls2-bmc: gen-ver: fix minor offset

The minor-version offset was incorrect, as it was set on `fw` instead of
the actual value, which made the build to crash:
line 25: printf: fw: invalid number

Change-Id: Ic74e5ef76292f74309b0be89e6ff708c372ff70c
---

diff --git a/package/hls2-bmc/gen-hls2-ver.sh b/package/hls2-bmc/gen-hls2-ver.sh
index bf901d5..9a0a1df 100755
--- a/package/hls2-bmc/gen-hls2-ver.sh
+++ b/package/hls2-bmc/gen-hls2-ver.sh
@@ -12,7 +12,7 @@
 	BUILD_DIR=${1}
 
 	MAJOR=$(cut -d- -f4 ${BUILD_DIR}/localversion-habanalabs)
-	MINOR=$(cut -d- -f6 ${BUILD_DIR}/localversion-habanalabs)
+	MINOR=$(cut -d- -f7 ${BUILD_DIR}/localversion-habanalabs)
 
 	arrMAJOR=(${MAJOR//./ })
 	arrMINOR=(${MINOR//./ })
